apply.sh

```
@@ -18,6 +18,9 @@ cd "$rootdir"
 ./keys/keys.sh || exit 1
 
 # base os
+printf "######## boot\n" 1>&2
+cd "$rootdir"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)